### ISSUE TITLE
Better sort articles for pass 2

### ISSUE THREAD

**Issue by davidskalinder** [davidskalinder/bpp-pass2-ui#6]
I think this is a question for the whole team, so I'll mention everybody: @olderwoman @limchaeyoon @matthewsmorgan @johnklemke

Sh/Could we sort articles by date and/or region to increase context for pass 2 coders?  Or has experience shown that this context isn't valuable enough?

I could swear I remember the team discussing this at some point in the past, but I can't find anything in the meeting notes, and of course the problem hasn't been directly noticeable to me until just now; so if we already know that this is a bad idea please somebody remind me and we should close this issue.

**Comment by olderwoman** [comment]
Yes, I originally thought that pass 2 would receive articles with events sorted by date and location. We forgot about that when Chaeyoon and Morgan wanted to get pass 2 ready for coder supervision. I’m going to ask John about this now.

**Comment by davidskalinder** [comment]
Having adopted the workflow of skipping difficult-to-code articles and therefore moving faster through the list of articles, I notice that after the first bunch of articles the sort order seems to become more natural (coder-date, maybe?  Or paper-date?  Or something like that).

So I don't know whether this has already been fixed or whether that's a happy accident of the order they were loaded into the DB or whether things will get better or worse later on in the queue (or where the queue is stored so I can investigate further).  But it does seem possible that all the jumps in the first few articles is an artifact of, or at least is intensified by, those articles being the hard-to-code leftovers of coders like me skipping them?

**Comment by olderwoman** [comment]
This does seem possible. The coders are reading articles in temporal order, so when you read work by one coder at a time it should be temporal
